teorija.tex

```diff
@@ -4069,9 +4068,9 @@ \subsection{Krog}
 Razmerje med obsegom in premerom kroga je \textbf{konstantno}. Konstanto označimo s~$\boldsymbol\pi$.
 \begin{align*}
   o &= 2\pi r \\
-  l &= \frac{\pi r\alpha}{180\deg} = r\alpha \\
+  l &= \frac{\pi r\alpha}{180\deg} = r\theta \\
   p &= \pi r^2 \\
-  p_{iz} &= \frac{\pi r^2\alpha}{360\deg} = \frac{r^2\alpha}{2} = \frac{l\krat
+  p_{iz} &= \frac{\pi r^2\alpha}{360\deg} = \frac{r^2\theta}{2} = \frac{l\krat
```

Comment:
Ne to da mi ni všec `\alpha` ampak sem hotel s `\thetho` nakazati, da je v radianih. Mogoče bi bilo bolje če bi dal zraven še komentar, da je drugi kot v radianih.